decentraland/unity-explorer
# feat: landscape update with remote manifest

## What does this PR change?

Fixes https://github.com/decentraland/unity-explorer/issues/493

## How to test the changes?

- No cache should be cleared during this test, the terrain should update automatically if any new scene is deployed.
- Teleport to `105,-113` and observe that there should be no terrain over the scene.
- Teleport to `-147,95` and observe that there should be no terrain over the scene.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
